.gitignore

```diff
@@ -0,0 +1,6 @@
+# .gitignore
+node_modules/
+
+linters.yml
```

Comment:
- [ ] Linters are not working on the Pull Request, kindly follow the instructions on the [documentation](https://github.com/microverseinc/linters-config). The linters.yml shouldn't be ignored for instance.